doc/sphinx/CMakeLists.txt

```diff
@@ -19,57 +19,49 @@ find_program(SPHINX_BUILD_EXECUTABLE sphinx-build REQUIRED)
 if(NOT SPHINX_BUILD_EXECUTABLE)
   message(FATAL_ERROR "sphinx-build not found")
 endif()
-find_program(SPHINX_APIDOC_EXECUTABLE sphinx-apidoc REQUIRED)
-if(NOT SPHINX_APIDOC_EXECUTABLE)
-  message(FATAL_ERROR "sphinx-apidoc not found")
+find_program(SPHINX_AUTOBUILD_EXECUTABLE sphinx-autobuild)
+if(NOT SPHINX_AUTOBUILD_EXECUTABLE)
+  message(STATUS "sphinx-autobuild not found")
+else()
+  message(STATUS "sphinx-autobuild found: ${SPHINX_AUTOBUILD_EXECUTABLE}")
 endif()
 find_program(JUPYTER_NBCONVERT_EXECUTABLE jupyter-nbconvert REQUIRED)
 if(NOT JUPYTER_NBCONVERT_EXECUTABLE)
   message(FATAL_ERROR "jupyter-nbconvert not found")
 endif()
 
 set(SPHINX_BASE "${CMAKE_CURRENT_BINARY_DIR}")
-
 set(BINARY_BUILD_DIR "${SPHINX_BASE}/_build")
 set(SPHINX_PYTHON_DIR "${CMAKE_BINARY_DIR}/pairinteraction")
 
 # Sphinx cache with pickled ReST documents
 set(SPHINX_CACHE_DIR "${SPHINX_BASE}/_doctrees")
 
 # HTML output directory
-set(SPHINX_HTML_DIR "${SPHINX_BASE}/html")
+set(SPHINX_HTML_DIR "${SPHINX_BASE}/_build/html")
 
 # Sphinx configuration file
 configure_file(
   "${CMAKE_CURRENT_SOURCE_DIR}/conf.py.in"
   "${SPHINX_BASE}/conf.py"
   @ONLY)
 
+# Directories to be copied to the build directory
+file(COPY "_static" DESTINATION "${SPHINX_BASE}")
+file(COPY "_templates" DESTINATION "${SPHINX_BASE}")
+
 # Prepare sphinx target
 add_custom_target(sphinx
-    ${CMAKE_COMMAND} -E env SPHINX_APIDOC_OPTIONS=members,inherited-members,show-inheritance
```

Comment:
Why was this removed?

Reply:
Once this has been adressed, please rebase and merge.

Reply:
In my tests, calling sphinx-apidoc explicitly was not needed to generate the documentation for the pairinteraction module. Admittedly, I am not completely sure whether this has been just the case on my systems or whether it is in general not needed. I could move the options "members, inherited-members, ..." to conf.py.in so that these options are still applied. What do you think?

Reply:
The API documentation looks fine to me but the nbsphinx gallery is broken.

Reply:
Once I encountered a similar issue with an old version of nbsphinx. I can try whether updating the version of nbsphinx in the CI solves the problem. If this works out, we could think about making a separate docker image with the newest nbsphinx version for building our documentation.